servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/WriteStreamSubscriber.java

```diff
@@ -340,9 +340,9 @@ private void requestMoreIfRequired(@Nullable Subscription subscription, long byt
          * </pre>
          * We assume that no listener for a write is added after that write is completed (a.k.a late listeners).
          * Most of the messages have 3 listeners (headers, one payload body chunk, trailers). Messages with large
-         * streaming payload body may have more listeners. However, the MIN_INITIAL_CAPACITY of ArrayDeque is 8.
```

Comment:
`MIN_INITIAL_CAPACITY` -> OpenJDK no longer has this restriction.

Reply:
nice! Do you know when this changed ?

Reply:
starting from JDK9, there is no `MIN_INITIAL_CAPACITY` anymore